spec/models/user_spec.rb

```diff
@@ -1,5 +1,22 @@
+# rubocop:disable all
+# spec/models/user_spec.rb
+
+
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'is valid with valid attributes' do
+    user = User.new(name: 'John', password: '3434', post_counter: 0)
+    expect(user).to be_valid
+  end
+
+  it 'is not valid without a name' do
+    user = User.new(post_counter: 0)
+    expect(user).not_to be_valid
+  end
+
+  it 'is not valid with a negative post counter' do
+    user = User.new(name: 'Alice', post_counter: -1)
+    expect(user).not_to be_valid
+  end
```

Comment:
- [ ] After running the currently existing tests, most of them were failing. So kindly ensure that all your tests are passing and kindly include the missing tests as well :+1: 
- Failed Tests
![image](https://github.com/SAZZAD1Q2/Gem-Web-Blog/assets/102699958/f1b36530-dec1-4061-9ee1-aa848d4799ae)

Reply:
Thank you for reviewing my project. But on my site I see no fail. Kindly will you inform me which command are you running to check my spec test case.

Reply:
HY @SAZZAD1Q2 I used the `rspec spec` command

Reply:
![Screenshot_191](https://github.com/SAZZAD1Q2/Gem-Web-Blog/assets/69638013/317fe36f-03ff-4e94-9bb4-f3b336963833)
-Tests are passing welldone 💪🏿